compiler-plugin/src/main/kotlin/arrow/meta/quotes/classorobject/ObjectDeclaration.kt

```diff
@@ -0,0 +1,38 @@
+package arrow.meta.quotes.classorobject
+
+import arrow.meta.quotes.Scope
+import org.jetbrains.kotlin.psi.KtObjectDeclaration
+
+/**
+ * <code>""" object $name { $body }""".objectDeclaration</code>
+ *
+ * TODO add wrappers for certain boolean logic so that there are more options to create KtObjectDeclaration i.e. companion object
+ * @see [DefaultElementScope::companionObject]
```

Comment:
```suggestion
 * @see [ElementScope::companionObject]
```
We should not link to the implementations but to the public APIs instead. The user is never exposed to `DefaultElementScope`.